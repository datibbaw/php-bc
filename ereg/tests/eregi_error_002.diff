004+ int(1)
005- Warning: eregi(): REG_EMPTY in %s on line %d
006+ Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
007+ 
008+ Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
010- Warning: eregi(): REG_EPAREN in %s on line %d
013+ Warning: preg_match(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
015- Warning: eregi(): REG_EBRACK in %s on line %d
018+ Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
020- Warning: eregi(): REG_EPAREN in %s on line %d
023+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
025- Warning: eregi(): REG_BADRPT in %s on line %d
028+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
030- Warning: eregi(): REG_BADRPT in %s on line %d
033+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
035- Warning: eregi(): REG_BADRPT in %s on line %d
038+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
040- Warning: eregi(): REG_BADRPT in %s on line %d
041- bool(false)
042+ bool(false)
044+ Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
045+ int(1)
046+ 
047+ Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
048+ int(1)
045- Warning: eregi(): REG_BADBR in %s on line %d
050- Warning: eregi(): REG_EMPTY in %s on line %d
052+ Warning: preg_match(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
055- Warning: eregi(): REG_EMPTY in %s on line %d
056+ bool(false)
057- 
058- Deprecated: Function eregi() is deprecated in %s on line %d
058+ Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
059+ 
060+ Warning: preg_match(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
065+ Warning: preg_match(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
067+ string(8) "original"
068+ Done
059- 
060- Warning: eregi(): REG_BADBR in %s on line %d
061- bool(false)
062- 
063- Deprecated: Function eregi() is deprecated in %s on line %d
064- 
065- Warning: eregi(): REG_ERANGE in %s on line %d
066- bool(false)
067- 
068- Deprecated: Function eregi() is deprecated in %s on line %d
069- 
070- Warning: eregi(): REG_EESCAPE in %s on line %d
071- bool(false)
072- 
073- Deprecated: Function eregi() is deprecated in %s on line %d
074- 
075- Warning: eregi(): REG_ERANGE in %s on line %d
076- bool(false)
077- string(8) "original"
078- Done