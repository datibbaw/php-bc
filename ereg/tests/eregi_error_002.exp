*** Testing eregi() : error conditions ***

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EBRACK in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_ERANGE in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EESCAPE in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_ERANGE in %s on line %d
bool(false)
string(8) "original"
Done