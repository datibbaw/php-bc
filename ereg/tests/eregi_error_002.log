
---- EXPECTED OUTPUT
*** Testing eregi() : error conditions ***

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EBRACK in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_ERANGE in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_EESCAPE in %s on line %d
bool(false)

Deprecated: Function eregi() is deprecated in %s on line %d

Warning: eregi(): REG_ERANGE in %s on line %d
bool(false)
string(8) "original"
Done
---- ACTUAL OUTPUT
*** Testing eregi() : error conditions ***

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
int(1)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
int(1)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
int(1)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)
string(8) "original"
Done
---- FAILED
