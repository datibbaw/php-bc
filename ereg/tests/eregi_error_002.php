<?php
/* Prototype  : proto int eregi(string pattern, string string [, array registers])
 * Description: Regular expression match 
 * Source code: ext/standard/reg.c
 * Alias to functions: 
 */

/*
 * Test bad regular expressions
 */

echo "*** Testing eregi() : error conditions ***\n";

$regs = 'original';

var_dump(eregi("", "hello"));
var_dump(eregi("c(d", "hello"));
var_dump(eregi("a[b", "hello"));
var_dump(eregi("c(d", "hello"));
var_dump(eregi("*", "hello"));
var_dump(eregi("+", "hello"));
var_dump(eregi("?", "hello"));
var_dump(eregi("(+?*)", "hello", $regs));
var_dump(eregi("h{256}", "hello"));
var_dump(eregi("h|", "hello"));
var_dump(eregi("h{0}", "hello"));
var_dump(eregi("h{2,1}", "hello"));
var_dump(eregi('[a-c-e]', 'd'));
var_dump(eregi('\\', 'x'));
var_dump(eregi('([9-0])', '1', $regs));

//ensure $regs unchanged
var_dump($regs);

echo "Done";
?>
