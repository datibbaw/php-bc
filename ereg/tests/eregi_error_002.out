*** Testing eregi() : error conditions ***

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
int(1)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
int(1)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
int(1)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)

Deprecated: Function eregi() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 108

Warning: preg_match(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 110
bool(false)
string(8) "original"
Done